config/200-clusterrole.yaml

```diff
@@ -65,3 +65,6 @@ rules:
   - apiGroups: ["networking.istio.io"]
     resources: ["virtualservices", "gateways"]
     verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
+  - apiGroups: ["certmanager.k8s.io"]
```

Comment:
Split this into a separate role (this isn't istio!) and label with `networking.knative.dev/certificate-provider: certmanager`